source/features/pr-branch-auto-delete.tsx

```diff
@@ -43,6 +43,16 @@ void features.add(import.meta.url, {
 	additionalListeners: [
 		onPrMerge,
 	],
+	awaitDomReady: true,
```

Comment:
Can we add a TODO comment saying that this can be removed after:

- https://github.com/refined-github/refined-github/issues/6566

Because then the check will be async (which asLongAs will await)